tests/validations/test_geometry_empty_check.py

```diff
@@ -0,0 +1,25 @@
+from geopackage_validator.utils import open_dataset
+from geopackage_validator.validations.geometry_empty_check import (
+    query_geometry_empty,
+    SQL_EMPTY_TEMPLATE,
+)
```

Comment:
Geen check/verify op een message zoals `Found empty geometry in table: {table_name}, column {column_name}, {count} {count_label}, example id {row_id`?

Reply:
Ja goeie, heb hier even puur de test voor `geometry_valid_check` als voorbeeld genomen, daarin wordt ook alleen de onderliggende query getest. Maar testen via de validator is eigenlijk beter inderdaad.